### PR TITLE
update makefile

### LINKED ISSUE #9: Create a Makefile to automate build
Automate the manual compile steps we did for #8

### PR DESCRIPTION
**Pull request by daphneherlambda**
Closes #9 
This Pr adds steps in makefile to compile the project

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,16 @@
 .PHONY: setup update run
 
-setup:
+setup-execution-helper:
```
Comment (jpcenteno) [inline Makefile:3]: IMHO, I would have these separated as different tasks. This is doing 3 distinct things

File: Makefile [hunk 2]
```diff
@@ -1,8 +1,16 @@
 .PHONY: setup update run
 
-setup:
+setup-execution-helper: 
+	cd ExecutionHelper && yarn install && yarn hardhat compile && yarn hardhat deploy-zksync --script deploy.ts
+
+setup-seaport:
```
Comment (jpcenteno) [inline Makefile:6]: Separate the setup from the compile steps

**Comment by fkrause98** [comment]
Looks good, but let's merge #10 first and adapt it here.

**Review by jpcenteno — CHANGES_REQUESTED** [review]

File: Makefile [hunk 3]
```diff
@@ -1,8 +1,22 @@
 .PHONY: setup update run
 
-setup:
+setup-execution-helper: 
+	cd ExecutionHelper && yarn install 
+
+compile-and-deploy-execution-helper:
+	cd ExecutionHelper && yarn hardhat compile && yarn hardhat deploy-zksync --script deploy.ts
```
Comment (fkrause98) [inline Makefile:7]: I wanted to reference this line

**Comment by jpcenteno** [comment]
- ✅ I was able to check that `make setup` works in a recently cloned repo.
- **TODO**  I haven't tested the `make run` command yet. Let me check that.

The code looks good to me. I would like to propose some improvements, which I can provide in another PR if you like.

File: Makefile [hunk 4]
```diff
@@ -1,8 +1,22 @@
 .PHONY: setup update run
```
Comment (jpcenteno) [inline Makefile:1]: We should remove this line and mark each one of the targets as PHONY

**Comment by jpcenteno** [comment]
**WAIT**

I tried to run calling `make run` first instead the `era-test-node` instance that I have running in the background at all times. This failed because it required to run `make setup` first, which depends on having the `era-test-node` running.

I'm going to make a PR pointing to this one with my suggested solution.

**Review by fkrause98 — APPROVED** [review]

**Review by jpcenteno — APPROVED** [review]
LGTM. Doing a follow-up PR #15

**Pull request merged**